web/styles/portico/markdown.css

```diff
@@ -352,12 +352,12 @@
     & kbd {
         /* Same as kbd in app_components.css */
         display: inline-block;
-        border: 1px solid hsl(0deg 0% 80%);
+        border: 1px solid var(--color-kbd-border);
         border-radius: 4px;
         font-weight: 600;
         white-space: nowrap;
-        background-color: hsl(0deg 0% 98%);
-        color: hsl(0deg 0% 20%);
+        background-color: var(--color-kbd-background);
+        color: var(--color-kbd-text);
```

Comment:
Are these variables set in portico pages? I think it'[s a bug to change this.

Reply:
You’re right. Karl’s [comment](https://github.com/zulip/zulip/pull/31655#discussion_r1771975883) on the other PR seems relevant here as well.